CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## v0.15 [2022-10-27]
+
+_What's new?_
+
+- New benchmarks added: ([#83](https://github.com/turbot/steampipe-mod-aws-thrifty/pull/83))([#85](https://github.com/turbot/steampipe-mod-aws-thrifty/pull/85))
```

Comment:
If each PR is adding a new benchmark, maybe we should move them to the lines below, e.g.,
```
  - Route 53 Checks (`steampipe check benchmark.route53`) ([#83](https://github.com/turbot/steampipe-mod-aws-thrifty/pull/83))
```

We can do a similar change for the controls below